cassandra/mp.py

```diff
@@ -52,21 +53,21 @@ def bootstrap_mp(argvals):
 
     if argvals.logdir is None:
         logdir = 'logs'
+        os.makedirs(logdir, exist_ok=True)
```

Comment:
I think we want to do this irrespective of whether we are using the default log location.

Reply:
Good point, I'll update it.